software/scholarsome.yml

```diff
@@ -0,0 +1,15 @@
+name: scholarsome
+website_url: https://www.scholarsome.com/
+description: Web-based and open source interactive flashcard learning software studying for the masses.
+licenses:
+  - GPL-3.0
+platforms:
+  - Node
+  - Docker
+tags:
+  - Learning and Courses
+source_code_url: https://github.com/hwgilbert16/scholarsome
+demo_url: https://scholarsome.com/
+stargazers_count: 505
+updated_at: '2024-07-23'
+archived: false
```

Comment:
```suggestion
```
these are redundant lines, they will be automatically maintained later 👍🏻

Reply:
Thank you for the advice, I fix that